org.eclipse.scout.rt.shared.test/src/test/java/org/eclipse/scout/rt/shared/http/AbstractHttpTransportManagerTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2010-2024 BSI Business Systems Integration AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     BSI Business Systems Integration AG - initial API and implementation
+ */
+package org.eclipse.scout.rt.shared.http;
+
+import static org.junit.Assert.*;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import org.junit.Test;
+
+import com.google.api.client.http.HttpRequestInitializer;
+
+public class AbstractHttpTransportManagerTest {
```

Comment:
How long does this test normally run? Would @SlowTest annotation make sense here?